test/scope-key-data-test.js

```diff
@@ -147,3 +147,16 @@ testHelpers.dev.devOnlyTest("Warn when key is not found and log the value of the
 	scopeKeyData.read();
 	teardown();
 });
+
+testHelpers.dev.devOnlyTest("Warn when key is not found and log the undefined value #206", function(assert) {
+	var teardown = testHelpers.dev.willWarn('Unable to find key "foo.length". Found "foo" with value: %o\n undefined');
+
+	var scope = new Scope({});
```

Comment:
I'm a little confused by this warning:

> Unable to find key "foo.length". Found "foo" with value: "undefined"

I would expect that to happen if the data looked like:

```js
{
  foo: undefined
}
```

If the data doesn't have a `foo` property, why do we want to say `"Found "foo" with value ..."`? We didn't find `foo`.